contrib/seeds/nodes_main.txt

```diff
@@ -630,1118 +747,166 @@
+[2a03:4000:51:33:64e5:a0ff:fe0f:60bd]:8333 # AS47147
+[2a03:4000:56:d:d40c:aaff:fe98:a555]:8333 # AS47147
+[2a03:4000:5d:bd4:a8bf:78ff:fe98:7ea4]:8333 # AS47147
+[2a03:4000:5d:ea7:944c:6bff:fead:b1f1]:8333 # AS47147
+[2a03:4000:5e:d78::31]:8333 # AS47147
+[2a03:4000:5f:cfc:14c3:eff:feb5:1c1a]:8333 # AS47147
+[2a03:6000:870:0:46:23:87:218]:8333 # AS51088
 [2a03:cfc0:8000:2a::9532:6516]:8333 # AS201814
-[2a03:cfc0:8000:2a::9532:651d]:8333 # AS201814
 [2a03:cfc0:8000:2a::9532:6520]:8333 # AS201814
+[2a03:cfc0:8000:2a::9532:6521]:8333 # AS201814
 [2a03:cfc0:8000:2a::9532:6522]:8333 # AS201814
 [2a03:cfc0:8000:2a::9532:6523]:8333 # AS201814
-[2a03:cfc0:8000:2a::9532:659a]:8333 # AS201814
-[2a03:cfc0:8000:2a::9532:659d]:8333 # AS201814
 [2a03:ec0:0:928::701:701]:8333 # AS199669
-[2a04:ee41:86:50b6:fa75:a4ff:fe3c:243f]:8333 # AS15796
-[2a05:d012:42a:5703:4dc5:8116:787c:e016]:8333 # AS16509
+[2a04:52c0:103:c455::1]:8333 # AS60404
+[2a04:52c0:104:160c::1]:8333 # AS60404
+[2a05:3580:d101:3700::]:8333 # AS20764
+[2a05:3580:dc0b:1600:6e46:5420:cd78:e7a]:8333 # AS20764
+[2a05:4cc0:0:332::2]:8333 # AS24750
+[2a05:6d40:b94e:d100:230:48ff:fedf:1432]:8333 # AS202128
 [2a05:d014:a55:4001:f6ab:dd5e:4039:b46c]:8333 # AS16509
 [2a05:d018:a75:6c00:c05b:4d0a:3658:1030]:8333 # AS16509
+[2a05:d01a:b7b:3c00:2d50:3236:f226:f15f]:8333 # AS16509
+[2a05:f480:2c00:100c:5400:4ff:fed7:dead]:8333 # AS20473
+[2a05:f480:3000:2b4e:5400:4ff:fed7:4206]:8333 # AS20473
+[2a06:dd01::36:0:0:1]:8333 # AS56694
+[2a07:7200:ffff:0:3016:d5ff:fe5e:1114]:8333 # AS34197
+[2a07:7200:ffff:0:b0a5:23ff:fe34:d292]:8333 # AS34197
 [2a07:9a07:3::2:1]:8333 # AS202605
-[2a07:d884::130e]:8333 # AS6762
-[2a0a:ef40:e44:9b01:2746:ca1e:6788:351c]:8333 # AS1273
+[2a07:b242:1000:1300:f250:8f0a:cdba:4d76]:8333 # AS25512
+[2a09:2681:1001::23]:8333 # AS61282
+[2a0a:31c0:100:0:888f:90ff:fe2c:761b]:8333 # AS6696
+[2a0a:4cc0:100:37b:c44a:2fff:fe10:2d3c]:8333 # AS47147
+[2a0a:4cc0:1:340:1460:fdff:feb2:2994]:8333 # AS47147
+[2a0b:f300:2:6::2]:8333 # AS62240
 [2a0b:f4c0:c1:920e:b25a:daff:fe87:77b4]:8333 # AS205100
+[2a0e:8f02:21d1:144::101]:8333 # AS20473
+[2a10:3781:2c19::1]:8333 # AS206238
+[2a10:3781:3a73:25::25]:8333 # AS206238
+[2a10:3781:3a73:25:a177:ad25:b14a:176a]:8333 # AS206238
 [2a10:c941:100:24::2:1001]:8333 # AS141011
+[2a11:d540:531:b00b::5]:8333 # AS6939
+[2a12:8ac1:a0::2900:2]:8333 # AS57797
+[2a12:8e40:5668:e410::1]:8333 # AS34465
+[2a12:8e40:5668:e412::1]:8333 # AS34465
+[2a12:8e40:5668:e415::1]:8333 # AS34465
+[2a12:8e40:5668:e417::1]:8333 # AS34465
+[2a12:8e40:5668:e418::1]:8333 # AS34465
+[2a12:8e40:5668:e41d::1]:8333 # AS34465
+[2a12:8e40:5668:e420::1]:8333 # AS34465
+[2a12:8e40:5668:e422::1]:8333 # AS34465
+[2a12:8e40:5668:e429::1]:8333 # AS34465
 [2a12:8e40:5668:f001::1]:8333 # AS34465
-[2a12:a302:1:a180::b5ca]:8333 # AS23959
-[2c0f:f4a8:b:b108:807d:b2d6:9146:38be]:8333 # AS37254
-[2c0f:f4a8:b:b108:c458:5c61:dcca:cb10]:8333 # AS37254
-iy7go4454pb4p2zmnkwrgsi6v6oqv53zxnmalz6rnfjemxftapfa.b32.i2p:0
```

Comment:
I hadn't regenerated to node lists because I wanted to wait until Ava's seeder resumed exporting good I2P node data. I have now pushed a preliminary export anyway. Everything should be there.